assets/js/Form.tsx

```diff
@@ -0,0 +1,81 @@
+import { useState } from 'react';
+import { Button, Input, InputGroup, InputRightElement, Stack } from '@chakra-ui/react'
+import React from 'react';
+
+type EventInfo = {
+  title: string;
+  description: string;
+  start_date: string;
+  presenters: string[];
+};
+
+type Links = {
+  viewer: string;
+  moderator: string;
+}
+
+function Form() {
+  const [links, setLinks] = useState<Links>();
+  const [currParticipant, setCurrParticipant] = useState<string>("");
+  const [eventInfo, setEventInfo] = useState<EventInfo>({title: "", description: "", start_date: "", presenters: []});
+
+  const handleInputTitle = (event: any) => setEventInfo({...eventInfo, title: event.target.value});
+  const handleInputDescription = (event: any) => setEventInfo({...eventInfo, description: event.target.value});
+  const handleInputDate = (event: any) => setEventInfo({...eventInfo, start_date: event.target.value});
+  const handleInputParticipant = (event: any) => setCurrParticipant(event.target.value);
+  const handleAddButton = () => {
+    setEventInfo({...eventInfo, presenters: [...eventInfo.presenters, currParticipant]});
+    setCurrParticipant("")
+  }
+
+  // ${window._env_.DASHBOARD_SERVER_URL}
+  const handleSendButton = () => {
+    fetch("http://localhost:4000/new", {
+      method: "post",
+      headers: { 
+        "Content-Type": "application/json",
+        'Accept': 'application/json'
+        },
+      body: JSON.stringify(eventInfo)
+    }).then((response) => {
+            return response.json();
+    }).then((data) => {
+      setLinks(JSON.parse(data));
+    });
+  };
```

Comment:
I think that we should make some validation, whether all required fields are set (in our case it will be only title and date), before sending the request.